Cargo.lock

```diff
@@ -881,7 +881,7 @@ dependencies = [
  "swc_ecma_transforms_typescript",
  "swc_ecma_utils",
  "swc_ecma_visit",
- "text_lines",
+ "text_lines 0.4.1",
```

Comment:
We have two of these now, but not a big deal because it's very small. I will fix this on the next deno_ast upgrade.